src/cheri/insns/gchi_32bit.adoc

```diff
@@ -20,6 +20,7 @@ include::wavedrom/gchi.adoc[]
 
 Description::
 Copy the metadata (bits [CLEN-1:MXLEN]) of capability `cs1` into `rd`.
+It is not required that the input capability `cs1`  has its tag set to 1.
```

Comment:
```suggestion
It is not required that the input capability `cs1`  has its valid tag set to 1.
```

ARC feedback was that we should always use valid(ity) tag to avoid confusion with future MTE tags.